apps/generator/docs/nunjucks-depreciate.md

```diff
@@ -0,0 +1,139 @@
+--
+title: "Migrating from Nunjucks to React render engine"
+weight: 170
+---
+
+## Migration Guide from nunjucks render engine to react render engine
+
+### Introduction
+
+The asyncAPI generator is moving away from Nunjucks templates in favor of React templates. This guide will help you migrate your existing Nunjucks templates to React. For a comprehensive understanding of why we introduced React as an alternative in 2021 and why we're now removing Nunjucks entirely, please read our article [React as a Generator Engine](https://www.asyncapi.com/blog/react-as-generator-engine). The principles discussed in this article still apply to our current transition.
+
+### Step-by-step migration guide
```

Comment:
probably because of my previous comments you will have to do changes to heading sizes, and start this from "##" and update the rest as well